src/editorial/web/components/lines/.babelrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	presets: [
+		"@babel/preset-env",
```

Comment:
It's possible that this is documented elsewhere and I've completely missed it, but how does `source` decide on the browser support list?

Reply:
There's nothing explicit yet. At the moment, it's whatever `preset-env` and ES5 gives us.

Reply:
It's an interesting one because we're targeting approximately three different environments (in the editorial stream):

- Node (10 for DCR, 12 for AR)
- Dotcom client-side (Safari, Firefox, Chrome, Edge, Internet Explorer etc.)
- Apps client-side (recent Safari and Chrome - webviews)

The support requirements for apps and dotcom on the client in particular are quite different. We _have_ to transpile down to the level of IE support for dotcom, but will that result in shipping redundant polyfills to the apps?

Reply:
Good points. Happy to pull this into a larger discussion around browser support.